### ISSUE TITLE
Excel import update to make somehting more usable for customers

### ISSUE THREAD

**Issue by phpgeekfr** [blockframes/blockframes#3816]
Here is a list of update to do:

- make FilmCode / internalRef optional (need to change some queries to check if movie is not already in db)
- split columns (year, prize => two columns) (no more `,` separator)
- 10 lines per title so we can add up to 10 directors for example (add colors (even/odd) to separate titles)  (no more `;` separator)
 ![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/7f5c8f17-db99-4855-bea5-306962d59520)
- update select list (genre, countries..)
- add validators on lists for cells : cell is red or greyed if not ok 
- add a custom columnt for "other" ( for example, genres must belong to a defined list but we can add a custom genre)
- check #3816 in code

**Comment by xmano** [comment]
On importing movie from excel, having some warnings, the warnings are not shown properly:

![image](https://user-images.githubusercontent.com/8731418/95322858-cbbc8100-08ba-11eb-9667-b9a184e89f0f.png)

**Comment by phpgeekfr** [comment]
Remark from Max: 
```
Hey just got an info from Francois that might concern you as well. He removed the totalBudget property from the movie and will set it in the campaign model, since we are only using it in Media Financiers
meaning, the excel import needs to be removed in terms of totalBudget
````

**Comment by lladreyt** [comment]
- [ ] When I download the excel sheet, my computer says it cannot open it because its format/extension is incorrect
<img width="1437" alt="Capture d’écran 2020-12-11 à 19 10 37" src="https://user-images.githubusercontent.com/73707444/101939059-b9293580-3be4-11eb-82b7-5c65724e2d8d.png">

**Comment by phpgeekfr** [comment]
@lladreyt Can you provide OS and browser name / version  and the software you used to open the file please ?

**Comment by Jackseed** [comment]
Closed for #4550
